README.md

```diff
@@ -17,6 +17,11 @@ containers as `hostPath` persistent storage objects.
   * `pv_annotations`: Dictionary with additional annotations for PV object
   * `pv_storage_class`: Storage class name
   * `pv_reclaim_policy`: Reclaim policy
+  * `encrypted`: Whether to setup the localstorage volume on a LUKS-on-LVM volume.
+     Encrypted volumes are skipped if the environment variable `ANSIBLE_OPENSHIFT_LOCALSTORAGE_<node>` is not set.
```

Comment:
`Encrypted volumes are skipped, if`?

Reply:
Comma is not required, cf. https://www.grammarly.com/blog/conditional-sentences/

> If the main clause precedes the if-clause, no punctuation is necessary.

Reply:
Good to know.